prowler/lib/outputs/compliance/mitre_attack.py

```diff
@@ -1,74 +1,91 @@
 from csv import DictWriter
+from importlib import import_module
 
 from colorama import Fore, Style
 from tabulate import tabulate
 
 from prowler.config.config import orange_color, timestamp
-from prowler.lib.outputs.compliance.models import Check_Output_MITRE_ATTACK
+from prowler.lib.logger import logger
 from prowler.lib.outputs.csv.csv import generate_csv_fields
 from prowler.lib.outputs.utils import unroll_list
 from prowler.lib.utils.utils import outputs_unix_timestamp
 
 
-def write_compliance_row_mitre_attack_aws(
+def write_compliance_row_mitre_attack(
```

Comment:
The `provider` object now has the `output_options` within it. You can remove one function argument if you want.

Reply:
Done